resources/user.rb

```diff
@@ -8,7 +8,4 @@
 attribute :database, kind_of: String
 attribute :connection, kind_of: Hash
 
-def initialize(*args)
-  super
-  @action = :add
-end
+default_action :add
```

Comment:
Foodcritic also complained about this one. It seems the default action is implicitly the first one specified which in this case would be 'add': https://docs.chef.io/custom_resources.html#default-action

I would prefer it explicitly defined, but ultimately it's up to you.